src/api/app/controllers/webui/user_controller.rb

```diff
@@ -195,28 +195,23 @@ def change_password
       return
     end
 
-    # check the valid of the params
-    unless User.current.password_equals?(params[:password])
-      errmsg = 'The value of current password does not match your current password. Please enter the password and try again.'
-    end
-    if params[:new_password] != params[:repeat_password]
```

Comment:
What happened with this check?

Reply:
Good catch! It's now implemented. See: https://github.com/openSUSE/open-build-service/pull/3814/files#diff-1b01dd7ca5b0e761374a4ec9dc902f4cR202

The rest is handled by `has_secure_password`.